BUILD.md

```diff
@@ -45,7 +45,7 @@ sudo apt install ffmpeg libsdl2-2.0-0
 # client build dependencies
 sudo apt install gcc git pkg-config meson ninja-build \
                  libavcodec-dev libavformat-dev libavutil-dev \
-                 libsdl2-dev
+                 libsdl2-dev adb
```

Comment:
`adb` is a runtime dependency (not a "build dependency"), so it should be added few lines above instead :wink:

Reply:
Ah, my apologies, I've just ammended it